docs/running-node-ansible/on.md

```diff
@@ -0,0 +1,89 @@
+# Running Observer Node using Ansible
+## Prerequisites
+Make sure you have all [prerequisites](./prerequisites.md) set up
+
+## Configure DCL network parameters (local machine)
+### 1. Set network chain id in [`deployment/ansible/inventory/hosts.yml`]
+```yaml
+all:
+  vars:
+    chain_id: <chain-id>
```

Comment:
In all the instructions from this folder, please explicitly specify what values must be set fort the current DCL Test Net (in addition to the abstract params/examples):
- chain_id
- genesis.json
- persistent_peers
- state sync params